app/uk/gov/hmrc/helptosavefrontend/controllers/RegisterController.scala

```diff
@@ -120,6 +131,23 @@ class RegisterController @Inject() (val messagesApi:           MessagesApi,
     }
   }(redirectOnLoginURL = FrontendAppConfig.checkEligibilityUrl)
 
+  private def checkIfAccountCreateAllowed(ifAllowed: ⇒ Future[Result])(implicit hc: HeaderCarrier) = {
+    helpToSaveService.isAccountCreationAllowed().fold(
```

Comment:
this can be shortened to:
```scala
helpToSaveService.isAccountCreationAllowed().fold(
      error ⇒ {
        logger.warn(s"Could not check if account create is allowed, due to: $error")
        ifAllowed
      },{ allowed ⇒
          if (allowed) {
            ifAllowed
          } else {
            SeeOther(routes.RegisterController.getUserCapReachedPage().url)
          }
      }
    )
```